<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SegmentationExplorer</class>
 <widget class="QWidget" name="SegmentationExplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Segmentation Explorer</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="groupLayout">
     <item>
      <widget class="QLabel" name="groupLabel">
       <property name="text">
        <string>Group by:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="groupList"/>
     </item>
     <item>
      <spacer name="groupSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="searchText">
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="placeholderText">
        <string>Search . . .</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearSearch">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Clear filters</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../rsc/espina.qrc">
         <normaloff>:/espina/cancel.png</normaloff>:/espina/cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Del</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CheckableTreeView" name="view">
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QTreeView::indicator {
     width: 15px;
     height: 15px;
 }
QTreeView::indicator:checked {
     image: url(:/espina/show_all.svg);
 } 
QTreeView::indicator:indeterminate {
     image: url(:/espina/show_partially.svg);
 } 
QTreeView::indicator:unchecked {
     image: url(:/espina/hide_all.svg);
 } 
</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::InternalMove</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="tagsLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Tags:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="selectedTags">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="controlLayout">
     <item>
      <spacer name="controlSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="specificControlLayout"/>
     </item>
     <item>
      <widget class="QPushButton" name="showInformationButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Show Information</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../rsc/espina.qrc">
         <normaloff>:/espina/info.svg</normaloff>:/espina/info.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Delete selected items</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../rsc/espina.qrc">
         <normaloff>:/espina/trash-full.svg</normaloff>:/espina/trash-full.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Del</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckableTreeView</class>
   <extends>QTreeView</extends>
   <header>GUI/Widgets/CheckableTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../rsc/espina.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>clearSearch</sender>
   <signal>clicked()</signal>
   <receiver>searchText</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
